src/app/[locale]/about/page.tsx

```diff
@@ -1,12 +1,8 @@
-import {useTranslations} from 'next-intl';
-import {PageHeader, PageTitle} from '@/components/page';
+import {useLocale} from 'next-intl';
 
-export default function About() {
-  const t = useTranslations('About');
+export default async function About() {
+  const locale = useLocale();
+  const AboutPage = (await import(`@/messages/${locale}/about.mdx`)).default;
```

Comment:
Curious: Is there a way to not have to use the `.default`?

Reply:
I don't think so. But I have moved this logic to one LocalizedMarkdown component, so it is simpler to use.